java/code/src/com/suse/manager/webui/controllers/DownloadController.java

```diff
@@ -515,8 +515,9 @@ private static void validateToken(String token, String channel, String filename)
                 halt(HttpStatus.SC_FORBIDDEN, "This token is not valid");
             }
         }, () -> {
-            LOG.info(String.format("Forbidden: token %s to access %s doesn't exists in the database", token, filename));
-            halt(HttpStatus.SC_FORBIDDEN, "This token is not valid");
+            LOG.debug(String.format(
+                    "Token %s to access %s doesn't exists in the database - could be an image build token",
+                    token, filename));
```

Comment:
I guess you can just use log4j formatting capabilities and save the `String.format()` if debug is disabled:
```suggestion
            LOG.debug("Token {} to access {} doesn't exists in the database - could be an image build token",
                token, filename);
```

Reply:
changed together with other problems